docs/ref-manual/cli/options.md

```diff
@@ -264,11 +264,13 @@ When different contracts have to be compiled for different Solidity versions.
 ### `--path`
 
 **What does it do?**  
-Use the given path as the root of the source tree instead of the root of the file system.  
+Passes the value of this option as it to the solc option `--allow-paths`
```

Comment:
`solc` should be code font.

Reply:
solc is not the right word, changed it to "solidity compiler"